internal/status/status.go

```diff
@@ -118,3 +120,30 @@ func WhoAMI(w http.ResponseWriter, req *http.Request) {
 	respByte, _ := json.MarshalIndent(&details, "", "  ")
 	w.Write(respByte)
 }
+
+func TrackUIMetric(w http.ResponseWriter, req *http.Request) {
+	w.Header().Add("content-type", "application/json; charset=utf-8")
+	resp := make(map[string]string)
+
+	metricName := req.FormValue("name")
+	resp["name"] = metricName
+
+	switch metricName {
+	case "buffer":
+		metrics.UIBufferCount.Inc()
+		resp["status"] = "ok"
```

Comment:
I think it's okay to just return an empty response with 200 status code, will save a tiny bit of traffic too.